integration/test/basic/basic_test.go

```diff
@@ -38,12 +34,16 @@ func TestChartLifecycle(t *testing.T) {
 		},
 	}
 
-	// Setup
-	l, err := micrologger.New(micrologger.Config{})
-	if err != nil {
-		t.Fatalf("could not create logger %v", err)
+	chartConfigValues := chartconfig.ChartConfigValues{
+		Channel:   "5-5-beta",
+		Name:      "tb-chart",
+		Namespace: "giantswarm",
+		Release:   "tb-release",
+		//TODO: fix this static VersionBundleVersion
+		VersionBundleVersion: "0.2.0",
```

Comment:
So why is it static at all? This looks super wrong. How do we make sure we update it with the operator evolving? In `cluster-operator` we get this from the installations repo, which is more generic. See https://github.com/giantswarm/cluster-operator/blob/3fbdc4529c6b3978a9e64d38a39bb3a838b31a2f/integration/setup/setup.go#L109-L138. The example there should be refactored as well because the setup there is not really generic and the templates there work with env vars and so on but at least it fetches the necessary versions.

Reply:
Short answer:
Because `chart-operator` isn't in the index blob yet. I want to do this in a separate PR, get chart-operator in there and then use this. The code in `cluster-operator` e2e tests uses e2e-harness to fetch the version information, we will need something like this in our operators as well. I am therefor targeting an implementation we can reuse there as well.

Reply:
Wasn't a short answer :trollface: but I get you. Cool.